diff -Naur wit.old/Makefile wit/Makefile
--- wit.old/Makefile	2013-02-15 14:15:14.000000000 +0100
+++ wit/Makefile	2013-02-15 14:19:17.917710371 +0100
@@ -172,18 +172,6 @@
 TOBJ_ALL	:= $(TOBJ_wit) $(TOBJ_wwt) $(TOBJ_wdf) $(TOBJ_wfuse)
 
 #-------------------------------------------------------------------------------
-# sub libs
-
-# libbz2
-LIBBZ2_SRC	= $(shell echo src/libbz2/*.c)
-LIBBZ2_OBJ	= $(patsubst %.c,%.o,$(LIBBZ2_SRC))
-
-# lib summary
-LIB_LIST	+= libbz2
-LIB_OBJECTS	+= $(LIBBZ2_OBJ)
-RM_FILES	+= $(foreach l,$(LIB_LIST),src/$(l)/*.{d,o})
-
-#-------------------------------------------------------------------------------
 # source files
 
 UI_FILES	= ui.def
@@ -211,7 +199,7 @@
 
 # other objects
 WIT_O		:= debug.o lib-std.o lib-file.o lib-sf.o \
-		   lib-bzip2.o lib-lzma.o \
+		   lib-lzma.o \
 		   lib-wdf.o lib-wia.o lib-ciso.o \
 		   ui.o iso-interface.o wbfs-interface.o patch.o \
 		   titles.o match-pattern.o dclib-utf8.o \
@@ -391,14 +379,6 @@
 
 #
 ###############################################################################
-# lib specific rules
-
-$(LIBBZ2_OBJ): %.o: %.c Makefile
-	@printf "$(LOGFORMAT)" object "$(subst src/libbz2/,,$@)" "$(MODE) [libbz2]"
-	@$(CC) $(CFLAGS) $(DEPFLAGS) $(DEFINES) -c $< -o $@
-
-#
-###############################################################################
 # specific rules in alphabetic order
 
 .PHONY : all
diff -Naur wit.old/src/lib-bzip2.h wit/src/lib-bzip2.h
--- wit.old/src/lib-bzip2.h	2013-02-15 14:15:10.000000000 +0100
+++ wit/src/lib-bzip2.h	2013-02-15 14:19:50.619352753 +0100
@@ -63,142 +63,6 @@
 
 //
 ///////////////////////////////////////////////////////////////////////////////
-///////////////			  helpers			///////////////
-///////////////////////////////////////////////////////////////////////////////
-
-ccp GetMessageBZIP2
-(
-    int			err,		// error code
-    ccp			unkown_error	// result for unkown error codes
-);
-
-//-----------------------------------------------------------------------------
-
-int CalcCompressionLevelBZIP2
-(
-    int			compr_level	// valid are 1..9 / 0: use default value
-);
-
-//-----------------------------------------------------------------------------
-
-u32 CalcMemoryUsageBZIP2
-(
-    int			compr_level,	// valid are 1..9 / 0: use default value
-    bool		is_writing	// false: reading mode, true: writing mode
-);
-
-//
-///////////////////////////////////////////////////////////////////////////////
-///////////////			BZIP2 writing			///////////////
-///////////////////////////////////////////////////////////////////////////////
-
-enumError EncBZIP2_Open
-(
-    BZIP2_t		* bz,		// data structure, will be initialized
-    File_t		* file,		// destination file
-    int			compr_level	// valid are 1..9 / 0: use default value
-);
-
-//-----------------------------------------------------------------------------
-
-enumError EncBZIP2_Write
-(
-    BZIP2_t		* bz,		// created by EncBZIP2_Open()
-    const void		* data,		// data to write
-    size_t		data_size	// size of data to write
-);
-
-//-----------------------------------------------------------------------------
-
-enumError EncBZIP2_Close
-(
-    BZIP2_t		* bz,		// NULL or created by EncBZIP2_Open()
-    u32			* bytes_written	// not NULL: store written bytes
-);
-
-//
-///////////////////////////////////////////////////////////////////////////////
-///////////////			BZIP2 reading			///////////////
-///////////////////////////////////////////////////////////////////////////////
-
-enumError DecBZIP2_Open
-(
-    BZIP2_t		* bz,		// data structure, will be initialized
-    File_t		* file		// source file
-);
-
-//-----------------------------------------------------------------------------
-
-enumError DecBZIP2_Read
-(
-    BZIP2_t		* bz,		// created by DecBZIP2_Open()
-    void		* buf,		// destination buffer
-    size_t		buf_size,	// size of destination buffer
-    u32			* buf_written	// not NULL: store bytes written to buf
-);
-
-//-----------------------------------------------------------------------------
-
-enumError DecBZIP2_Close
-(
-    BZIP2_t		* bz		// NULL or created by DecBZIP2_Open()
-);
-
-//
-///////////////////////////////////////////////////////////////////////////////
-///////////////		    BZIP2 memory conversions		///////////////
-///////////////////////////////////////////////////////////////////////////////
-
-enumError EncBZIP2buf
-(
-    void		*dest,		// valid destination buffer
-    uint		dest_size,	// size of 'dest'
-    uint		*dest_written,	// store num bytes written to 'dest', never NULL
-
-    const void		*src,		// source buffer
-    uint		src_size,	// size of source buffer
-
-    int			compr_level	// valid are 1..9 / 0: use default value
-);
-
-//-----------------------------------------------------------------------------
-
-enumError EncBZIP2
-(
-    u8			**dest_ptr,	// result: store destination buffer addr
-    uint		*dest_written,	// store num bytes written to 'dest', never NULL
-    bool		use_iobuf,	// true: allow thhe usage of 'iobuf'
-
-    const void		*src,		// source buffer
-    uint		src_size,	// size of source buffer
-
-    int			compr_level	// valid are 1..9 / 0: use default value
-);
-
-//-----------------------------------------------------------------------------
-
-enumError DecBZIP2buf
-(
-    void		*dest,		// valid destination buffer
-    uint		dest_size,	// size of 'dest'
-    uint		*dest_written,	// store num bytes written to 'dest', never NULL
-
-    const void		*src,		// source buffer
-    uint		src_size	// size of source buffer
-);
-
-//-----------------------------------------------------------------------------
-
-enumError DecBZIP2
-(
-    u8			**dest_ptr,	// result: store destination buffer addr
-    uint		*dest_written,	// store num bytes written to 'dest', never NULL
-    const void		*src,		// source buffer
-    uint		src_size	// size of source buffer
-);
-
-//
-///////////////////////////////////////////////////////////////////////////////
 ///////////////				END			///////////////
 ///////////////////////////////////////////////////////////////////////////////
 
